src/VisualStudio/Core/Def/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs

```diff
@@ -219,7 +169,7 @@ void computeDiagnosticsToFix(IUIThreadOperationContext context)
             // snapshots. Otherwise, get all diagnostics from the diagnostic service.
             var diagnosticsToFixTask = selectedEntriesOnly
                 ? _suppressionStateService.GetSelectedItemsAsync(isAddSuppression, cancellationToken)
-                : GetAllBuildDiagnosticsAsync(shouldFixInProject, cancellationToken);
+                : Task.FromResult<ImmutableArray<DiagnosticData>>([]);
```

Comment:
didn't feel comfortable removing a ton more things here for servicing.  can re-evaluate in main how much (if any) of this type is needed.